packages/terra-graphs-docs/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 0.0.1 - (November 30, 2021)
```

Comment:
this should also be `1.0.0`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/274/files#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1R5).